solr/core/src/java/org/apache/solr/util/SolrVersion.java

```diff
@@ -32,7 +32,7 @@ public final class SolrVersion implements Comparable<SolrVersion> {
   private final Version version;
 
   // This static variable should be bumped for each release
-  private static final String LATEST_STRING = "10.0.0";
+  public static final String LATEST_STRING = "10.0.0";
 
   /** This instance represents the current (latest) version of Solr. */
   public static final SolrVersion LATEST = SolrVersion.valueOf(LATEST_STRING);
```

Comment:
Would it be better to just use `LATEST` since its already public? I know it would be converting back to string potentially but keeps the encapsulation that was here.

Reply:
I tried this initially, but since this is an annotation value we're specifying, the compiler complains: `Attribute value must be constant`